Index: jogl-v2.1.4/make/joglversion
===================================================================
--- jogl-v2.1.4.orig/make/joglversion	2014-02-11 15:56:19.285429247 +0100
+++ jogl-v2.1.4/make/joglversion	2014-02-11 15:57:01.845428206 +0100
@@ -2,7 +2,7 @@
 Application-Name: Java Bindings for OpenGL
 Specification-Title: Java Bindings for OpenGL API Specification
 Specification-Version: @BASEVERSION@
-Specification-Vendor: JogAmp Community
+Specification-Vendor: JogAmp Community - Debian package
 Implementation-Title: Java Bindings for OpenGL Runtime Environment
 Implementation-Version: @VERSION@
 Implementation-Build: @BUILD_VERSION@
@@ -15,5 +15,6 @@
 Trusted-Library: true
 Permissions: all-permissions
 Application-Library-Allowable-Codebase: *
+Class-Path: /usr/share/java/gluegen2-rt.jar
 @JAR_CODEBASE_TAG@
 
